docs/UserGuide.md

```diff
@@ -43,10 +49,13 @@ _More details coming soon ..._
 
 Adds a new opening to the list.
 
-Format: `add /n POSITION /c COMPANY /s STATUS /d DEADLINE`
+Format: `add p/POSITION n/NAME_OF_COMPANY c/CONTACT_NUMBER e/EMAIL s/STATUS [d/KEY_DATES]…​`
+
+* `KEY_DATES` must be in the form `EVENT@DATE`, where `EVENT` is a stage in the application process and `DATE` is the date of the stage in `YYYYMMDD` format.
+* `STATUS` **must be of either ongoing, successful or rejected**.
```

Comment:
Should we have more options?
The model I implemented has FOUND, APPLIED, INTERVIEWING, OFFERED, ACCEPTED, REJECTED

Reply:
Ok I will change it to match the implementation.